### PR TITLE
PT-1351, VIR-9: Resend notification button is added to the journal

### PR DESCRIPTION
**Pull request by getmansky**
Original project task is [VIR-9](https://virtocommerce.atlassian.net/secure/RapidBoard.jspa?rapidView=101&modal=detail&selectedIssue=VIR-9).

### CODE REVIEW

**Comment by mvktsk** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-7365 has been created

File: src/VirtoCommerce.NotificationsModule.Web/Scripts/blades/notifications-journal.js [hunk 1]
```diff
@@ -67,6 +67,23 @@ angular.module('virtoCommerce.notificationsModule')
                 name: "platform.commands.refresh", icon: 'fa fa-refresh',
                 executeMethod: blade.refresh,
                 canExecuteMethod: function () { return true; }
+            },
+            {
+                name: 'notifications.blades.notification-journal.labels.resend-messages',
+                icon: 'fa fa-repeat',
+                executeMethod: function (blade) {
```
Comment (Vectorfield4) [inline src/VirtoCommerce.NotificationsModule.Web/Scripts/blades/notifications-journal.js:74]: The blade is already declared.

```suggestion
                executeMethod: function () {
```

File: src/VirtoCommerce.NotificationsModule.Web/Scripts/blades/notifications-journal.js [hunk 2]
```diff
@@ -67,6 +67,23 @@ angular.module('virtoCommerce.notificationsModule')
...
+                    var messageIds = blade.$scope.gridApi.grid.rows
+                        .filter(x => x.isSelected)
+                        .map(x => x.entity.id);
+
+                    notifications.resendNotifications(messageIds, function () {
+                            blade.refresh();
+                        });
+                },
+                canExecuteMethod: function (blade) {
```
Comment (Vectorfield4) [inline src/VirtoCommerce.NotificationsModule.Web/Scripts/blades/notifications-journal.js:83]: The blade is already declared.

```suggestion
                canExecuteMethod: function () {
```

**Review by Vectorfield4 — CHANGES_REQUESTED** [review]
Hi! Thanks for your contribution! Please, fix code smells :)

**Review by akak1977 — APPROVED** [review]
Looks ok, but need to run/check by QA.

**Comment by Vectorfield4** [comment]
Waiting for fixed to create an alpha build.

**Review by Vectorfield4 — APPROVED** [review]

**Comment by Vectorfield4** [comment]
@getmansky alpha build https://vc3prerelease.blob.core.windows.net/packages/VirtoCommerce.Notifications_3.16.0-alpha.180.zip

**Pull request merged**